pkgs/applications/emulators/citra/generic.nix

```diff
@@ -87,17 +112,20 @@ stdenv.mkDerivation rec {
     echo 'set(BUILD_FULLNAME "${branchCaptialized} ${version}")' >> CMakeModules/GenerateBuildInfo.cmake
 
     # Devendoring
-    rm -rf externals/zstd externals/libressl
+    rm -rf externals/zstd
     cp -r ${zstd.src} externals/zstd
-    tar xf ${libressl.src} -C externals/
-    mv externals/${libressl.name} externals/libressl
     chmod -R a+w externals/zstd
   '';
 
-  # Fixes https://github.com/NixOS/nixpkgs/issues/171173
-  postInstall = lib.optionalString (enableCubeb && enableSdl2) ''
```

Comment:
I don't know whether the Cubeb libraries can actually be removed from here, since the SDL frontend segfaults on my end.

Reply:
Since the SDL backend only segfaults on OpenGL, I could test on Vulkan that Cubeb works (now that it is a proper dependency).